mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -314,6 +314,14 @@
     <string name="preferences_add_private_browsing_shortcut">Add private browsing shortcut</string>
     <!-- Preference for enabling "HTTPS-Only" mode -->
     <string name="preferences_https_only_title">HTTPS-Only Mode</string>
+
+    <!-- Description of the cookie banner reduction setting. -->
+    <string name="preferences_cookie_banner_reduction">Cookie Banner Reduction</string>
```

Comment:
Could you please explain in comment what this feature is? To give localizers context in order to translate it

Reply:
Updated in https://github.com/mozilla-mobile/fenix/pull/27930.